src/ClientData/FunctionUtils.cpp

```diff
@@ -63,7 +63,13 @@ std::optional<uint64_t> GetAbsoluteAddress(const orbit_client_protos::FunctionIn
 
   CHECK(!base_addresses.empty());
 
-  return func.address() + base_addresses.at(0) - module.load_bias();
+  auto result = func.address() + base_addresses.at(0) - module.load_bias() -
+                module.executable_segment_offset();
+  LOG("Calculating absolute address for %s: %#x(function_address) + %#x(base_address) - "
+      "%#x(load_bias) - %#x(segment_offset) = %#x",
+      func.pretty_name(), func.address(), base_addresses.at(0), module.load_bias(),
+      module.executable_segment_offset(), result);
```

Comment:
How often is this function called? Doesn't this spam the logs? This looks more intended for development.

Reply:
ops - this was debugging leftover log - removed.